clients/js/alph.ts

```diff
@@ -21,24 +19,25 @@ import {
 export async function executeGovernanceAlph(
   payload: GovernancePayload,
   vaa: Buffer,
-  network: 'MAINNET' | 'TESTNET' | 'DEVNET'
+  network: 'MAINNET' | 'TESTNET' | 'DEVNET',
+  nodeUrl?: string
 ) {
-  const n = NETWORKS[network]['alephium']
-  if (!n.rpc) {
+  const n = CONFIGS[network]['alephium']
```

Comment:
[Nit]maybe rename: `network -> networkType` and `n -> network`

But it seem single-character variable name is used sometimes in the codebase.